tests/installation/installation_overview.pm

```diff
@@ -21,7 +21,7 @@ sub run() {
 
     # overview-generation
     # this is almost impossible to check for real
-    assert_screen "inst-overview";
+    assert_screen "installation-settings-overview-loaded";
```

Comment:
with this removed we now never check for the actual tag 'inst-overview' within the test module 'installation_overview' which is at least … curious. WDYT?